### PR TITLE
Translate Pod Lifecycle to Bahasa Indonesia

### PR DESCRIPTION
**Pull request by wahyuoi**
Addresses #15341 
/assign @irvifa

### CODE REVIEW

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+Untuk informasi lebih mendalam mengenai status Pod dan kontainer, silakan lihat 
+[PodStatus](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podstatus-v1-core)
+dan 
+[ContainerStatus](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#containerstatus-v1-core).
+Mohon diperhatikan, informasi tentang status Pod bergantung pada 
+[ContainerState](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#containerstatus-v1-core).
+
+## State Kontainer
+
+Ketika Pod sudah ditempatkan pada suatu _node_ oleh scheduler, kubelet mulai membuat kontainer menggunakan container runtime.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
Ketika Pod sudah ditempatkan pada suatu _node_ oleh scheduler, kubelet mulai membuat kontainer menggunakan _runtime_ kontainer. 
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+* `Running`: Menandakan kontainer telah berjalan tanpa masalah. Setelah kontainer masuk ke _state_ Running, jika terdapat _hook_ `postStart` maka akan dijalankan. _State_ ini juga menampilkan waktu ketika kontainer masuk ke _state_ Running.
+
+   ```yaml
+   ...
+      State:          Running
+       Started:      Wed, 30 Jan 2019 16:46:38 +0530
+   ...
+   ```   
+       
+* `Terminated`:  Menandakan kontainer telah menyelesaikan eksekusi dan telah berhenti berjalan. Kontainer akan menjadi _state_ ini ketika telah menyelesaikan eksekusi atau terjadi kesalahan. Terlepas dari itu, sebuah alasan dan _exit code_ akan ditampilkan, bersama dengan waktu kontainer mulai dijalankan dan waktu berhenti. Sebelum kontainer masuk ke _state_ Terminated, jika terdapat `preStop` _hook_ maka akan dijalankan.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
* `Terminated`:  Menandakan kontainer telah menyelesaikan "tugasnya". Kontainer akan menjadi _state_ ini ketika telah menyelesaikan eksekusi atau terjadi kesalahan. Terlepas dari itu, sebuah alasan dan _exit code_ akan ditampilkan, bersama dengan waktu kontainer mulai dijalankan dan waktu berhenti. Sebelum kontainer masuk ke _state_ Terminated, jika terdapat `preStop` _hook_ maka akan dijalankan.
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+      lastProbeTime: null
+      lastTransitionTime: 2018-01-01T00:00:00Z
+  containerStatuses:
+    - containerID: docker://abcd...
+      ready: true
+...
+```
+
+Kondisi Pod yang baru harus memenuhi [format label](/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) pada Kubernetes.
+Sejak perintah `kubectl patch` belum mendukung perubahan status obyek, kondisi Pod yang baru harus mengubah melalui aksi `PATCH` dengan menggunakan
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
Sejak perintah `kubectl patch` belum mendukung perubahan status objek, kondisi Pod yang baru harus mengubah melalui aksi `PATCH` dengan menggunakan
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+- Menggunakan sebuah [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) untuk Pod yang akan berjalan
+  hanya satu untuk setiap mesin, karena menyediakan servis yang spesifik untuk suatu mesin.
+
+
+Ketiga tipe pengontrol ini memiliki sebuah PodTemplate. Direkomdasikan untuk membuat 
+pengontrol yang sesuai dan membiarkan ini membuat Pod, daripada membuat Pod sendiri secara langsung.
+Karena Pod itu sendiri tidak tahan terhadap gagalnya suatu mesin, namun pengontrol tahan.
+
+Jika _node_ mati atau sambungannya terputus dari _cluster_, Kubernetes mengatur
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
Jika node mati atau sambungannya terputus dari kluster, Kubernetes mengatur
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+`phase` dari semua Pod pada _node_ yang mati untuk menjadi Failed.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
`phase` dari semua Pod pada node yang mati untuk menjadi Failed.
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+
+## Contoh
+
+### Contoh Liveness Probe tingkat lanjut
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
### Contoh _Liveness Probe_ tingkat lanjut
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+
+_Liveness probes_ dieksekusi oleh kubelet, jadi semua permintaan akan dilakukan
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
_Liveness probe_ dieksekusi oleh kubelet, jadi semua permintaan akan dilakukan 
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+        port: 8080
+        httpHeaders:
+        - name: X-Custom-Header
+          value: Awesome
+      initialDelaySeconds: 15
+      timeoutSeconds: 1
+    name: liveness
+```
+
+### Contoh _states_
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
### Contoh _State_
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+[ContainerState](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#containerstatus-v1-core).
+
+## State Kontainer
+
+Ketika Pod sudah ditempatkan pada suatu _node_ oleh scheduler, kubelet mulai membuat kontainer menggunakan container runtime. 
+Ada tiga kemungkinan _state_ untuk suatu kontainer, yaitu Waiting, Running, dan Terminated.
+Untuk mengecek _state_ suatu kontainer, kamu bisa menggunakan perintah `kubectl describe pod [NAMA_POD]`.
+_State_ akan ditampilkan untuk masing-masing kontainer dalam Pod tersebut.
+
+* `Waiting`: Merupakan _default state_ dari kontainer. Jika _state_ kontainer bukan Running atau Terminated, berarti dalam _Wating state_.
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
* `Waiting`: Merupakan _state_ default dari kontainer. Jika _state_ kontainer bukan Running atau Terminated, berarti dalam _Wating state_. 
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+_endpoint_ tersebut berbeda dengan _endpoint_ untuk pengecekan _liveness_.
+
+Perlu dicatat, jika kamu hanya ingin bisa menutup _request_ ketika Pod sedang dihapus
+maka kamu tidak perlu menggunakan pemeriksaan _readiness_. Dalam penghapusan, Pod akan
+secara otomatis mengubah _state_ dirinya menjadi _unready_ tanpa peduli apakah terdapat
+pemeriksaan _readiness_ atau tidak. Pod tetap ada pada _state unready_ selama menunggu 
+kontainer dalam Pod berhenti.
+
+Untuk informasi lebih lanjut mengenai pengaturan pemeriksaan _liveness_ atau _readiness_, lihat bagian
+[Configure Liveness and Readiness Probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
[Konfigurasi _Liveness_ dan _Readiness_ _Probe_](/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
```

File: content/id/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 11]
```diff
@@ -0,0 +1,353 @@
+---
+title: Siklus Hidup Pod
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+{{< comment >}}Pembaruan: 4/14/2015{{< /comment >}}
+{{< comment >}}Diubah dan dipindahkan ke bagian _Concepts_: 2/2/17{{< /comment >}}
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod-lifecycle.md]:
```suggestion
{{< comment >}}Diubah dan dipindahkan ke bagian konsep: 2/2/17{{< /comment >}}
```
The section in Bahasa should be konsep https://kubernetes.io/id/docs/concepts/

**Comment by irvifa** [comment]
Merged 🎉 thanks a lot!

**Comment by wahyuoi** [comment]
Thanks @irvifa :bowing_man:

**Pull request merged**